kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/MctsBeneficiaryImportReaderServiceImpl.java

```diff
@@ -32,33 +31,38 @@
 import java.io.BufferedReader;
 import java.io.IOException;
 import java.io.Reader;
-import java.util.HashMap;
+import java.util.List;
+import java.util.ArrayList;
 import java.util.Map;
-
-import static org.motechproject.nms.kilkari.utils.RejectedObjectConverter.childRejectionRch;
-import static org.motechproject.nms.kilkari.utils.RejectedObjectConverter.convertMapToRchChild;
+import java.util.HashMap;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
 
 @Service("mctsBeneficiaryImportReaderService")
 public class MctsBeneficiaryImportReaderServiceImpl implements MctsBeneficiaryImportReaderService {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(MctsBeneficiaryImportReaderServiceImpl.class);
+    private static final int RECORDS_PART_SIZE = 10000;
```

Comment:
Comment ! What size is this ! - can this not be moved to some constants file/properties file -- since this might depend on system